auth/gcloud/aio/auth/token.py

```diff
@@ -324,8 +355,9 @@ async def _refresh_service_account(self, timeout: int) -> TokenResponse:
             timeout=timeout,
         )
         content = await resp.json()
-        return TokenResponse(value=str(content['access_token']),
-                             expires_in=int(content['expires_in']))
+        token = str(content.get('access_token') or content.get('id_token'))
```

Comment:
I think we want a KeyError if neither of these exist, since AFAIK that would still be invalid. Something like this?
```suggestion
        token = str(content.get('access_token') or content['id_token'])
```